packages/language-server/src/plugins/typescript/features/DocumentSymbolsProvider.ts

```diff
@@ -46,15 +48,18 @@ export class DocumentSymbolsProviderImpl implements DocumentSymbolsProvider {
 				Range.create(
 					document.positionAt(document.astroMeta.frontmatter.endOffset ?? 0),
 					document.positionAt(document.getTextLength())
-				)
+				),
+				document.getURL()
 			)
 		);
 
 		for (let symbol of symbols.splice(1)) {
 			symbol = mapSymbolInformationToOriginal(fragment, symbol);
 
 			if (document.offsetAt(symbol.location.range.end) >= (document.astroMeta.content.firstNonWhitespaceOffset ?? 0)) {
-				symbol.containerName = 'Template';
+				if (symbol.containerName === originalContainerName) {
```

Comment:
Symbols not at the root of the template previously had the wrong containerName. I don't think this caused any issues however